== SUBMITION (2019-05-14 16:37:19) 
-- CODE:
x= eval(input("valor de x: "))
k= int(input("n de k: "))
contador= 1 
sinal= -1
resultado= x 
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/ factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2 
	contador +=1
	
print(round(soma, 10))
-- EXECUTION TIME:
0.249582
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x= eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-14 16:37:26) 
-- CODE:
x= eval(input("valor de x: "))
k= int(input("n de k: "))
contador= 1 
sinal= -1
resultado= x 
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/ factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2 
	contador +=1
	
print(round(soma, 10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    x= eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:39:00) 
-- CODE:
x= eval(input("valor de x: "))
k= int(input("n de k: "))
contador= 1 
sinal= -1
resultado= x 
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2 
	contador +=1
	
print(round(soma, 10))
-- EXECUTION TIME:
0.14647
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 1, in <module>
    x= eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-14 16:39:22) 
-- CODE:
x= eval(input("valor de x: "))
k= int(input("n de k: "))
contador= 1 
sinal= -1
resultado= x 
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2 
	contador +=1
	
print(round(soma, 10))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    x= eval(input())
  File "<string>", line 1, in <module>
NameError: name 'pi' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:43:47) 
-- CODE:

from math import*


x= eval(input("valor de x: "))
k= int(input("n de k: "))
contador= 1 
sinal= -1
resultado= x 
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2 
	contador +=1
	
print(round(soma, 10))
-- EXECUTION TIME:
0.166878
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1716/2505/main.py", line 18, in <module>
    print(round(soma, 10))
NameError: name 'soma' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-14 16:44:33) 
-- CODE:

from math import*
x= eval(input("valor de x: "))
k= int(input("n de k: "))
contador= 1 
sinal= -1
resultado= x 
expoente= 3

while(contador < k):
	resultado= resultado + (sinal*(x**expoente/factorial(expoente)))
	sinal= sinal*(-1)
	expoente= expoente + 2 
	contador +=1
	
print(round(resultado, 10))
-- EXECUTION TIME:
0.122392
-- TEST CASE 1:
---- input:
pi/2
5
---- correct output:
1.0000035426
---- user output:
1.0000035426
-- TEST CASE 2:
---- input:
-pi/2
4
---- correct output:
-0.9998431014
---- user output:
-0.9998431014
-- TEST CASE 3:
---- input:
pi/3
3
---- correct output:
0.8662952838
---- user output:
0.8662952838
-- GRADE:
100%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
